4.8/runtime/windowsservercore-2004/Dockerfile

```diff
@@ -10,6 +10,8 @@ ENV `
 
 RUN `
     # Ngen top of assembly graph to optimize a set of frequently used assemblies
-    \Windows\Microsoft.NET\Framework64\v4.0.30319\ngen install "Microsoft.PowerShell.Utility.Activities, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35"
+    \Windows\Microsoft.NET\Framework64\v4.0.30319\ngen install "Microsoft.PowerShell.Utility.Activities, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" `
+    && \Windows\Microsoft.NET\Framework64\v4.0.30319\ngen update
```

Comment:
This isn't needed.  It amounts to a no-op.  Everything is already up-to-date with ngen at this point.

```
C:\>docker run --rm mcr.microsoft.com/dotnet/framework/runtime:4.8-windowsservercore-2004 cmd /c \Windows\Microsoft.NET\Framework64\v4.0.30319\ngen update /verbose
Microsoft (R) CLR Native Image Generator - Version 4.8.4084.0
Copyright (c) Microsoft Corporation.  All rights reserved.
All compilation targets are up to date.
```